pageFactory/objectRepository/AssetPageObjects.ts

```diff
@@ -0,0 +1,212 @@
+import { Page,Locator } from "@playwright/test";
+export class AssetPageObjects{
+    constructor(page:Page){
+        this.page=page
+    }
+    public page:Page
```

Comment:
Before pushing your changes make sure every page should be formatted

Reply:
Every page is formatted